documentation/docs/installation/wallet.md

```diff
@@ -2,4 +2,30 @@
 
 ## Why does Hummingbot need my Ethereum wallet private key?
 
-Strategies that transact on Decentralized Exchanges (such as Radar Relay and DDEX) are direct interactions with that exchange’s smart contracts on the Ethereum blockchain.  Therefore, transactions must be signed and authorized, which requires your private key.
\ No newline at end of file
+Strategies that transact on decentralized exchanges (such as [Radar Relay](/connectors/radar-relay) and [DDEX](/connectors/ddex)) are direct interactions with that exchange’s smart contracts on the Ethereum blockchain.  Therefore, transactions must be signed and authorized, which requires your private key.
```

Comment:
suggestion - Strategies that transact on decentralized exchanges (such as [Radar Relay](/connectors/radar-relay) and [DDEX](/connectors/ddex)) directly interact with smart contracts on the Ethereum blockchain.